rebasehelper/patch_helper.py

```diff
@@ -179,7 +179,7 @@ def _git_rebase(cls):
                     with open(base_name, 'wb') as f:
                         f.write(diff)
                         f.write(b'\n')
-                    modified_patches.append(base_name)
+                    modified_patches.append(patch_name)
```

Comment:
This is not good. Sure, it fixes the issue, but it doesn't belong to this commit and it's not explained anywhere.